types/simulation/types.go

```diff
@@ -137,7 +139,12 @@ type AppParams map[string]json.RawMessage
 // case of operation weights where Rand is not used).
 func (sp AppParams) GetOrGenerate(cdc codec.JSONMarshaler, key string, ptr interface{}, r *rand.Rand, ps ParamSimulator) {
 	if v, ok := sp[key]; ok && v != nil {
-		cdc.MustUnmarshalJSON(v, ptr)
+		msg, ok := ptr.(proto.Message)
+		if !ok {
+			panic(fmt.Errorf("can't proto marshal %T", ptr))
+		}
+
+		cdc.MustUnmarshalJSON(v, msg)
```

Comment:
Does this need protobuf or can we use `encoding/json`?

Reply:
`encoding/json` seems to be ok indeed.